### ISSUE TITLE
Swing timer on Hunter

### ISSUE THREAD

**Issue by Freeruns** [WeakAuras/WeakAuras2#1634]
**Describe the bug**

Swing timer trigger doesnt work on hunters

**Comment by mrbuds** [comment]
Can you give more information? Which version of weakauras ? Which version of wow ? Example with aura included ?

**Comment by Freeruns** [comment]
2.14.5 version
2.14.2 - work great
example wa: https://pastebin.com/W8UsnYJv

**Comment by mrbuds** [comment]
Before changing the trigger for classic i asked a few hunter theorycrafter if they have any use for a swing timer to optimize their rotation on retail, as the previous code wasn't making any sense, timer started on HIT instead of on SHOOT, and they all said it's not relevant for any hunter spec, so this was dropped with 2.14.
TLDR: it was broken, and wasn't used, now it's gone

**Comment by Freeruns** [comment]
RLY? i guess this hunters are stupid. Because we have talent and swing
timer is important for tracking procs

чт, 29 авг. 2019 г. в 19:17, Buds <notifications@github.com>:

**Comment by simhavaktra** [comment]
> Before changing the trigger for classic i asked a few hunter theorycrafter if they have any use for a swing timer to optimize their rotation on retail, as the previous code wasn't making any sense, timer started on HIT instead of on SHOOT, and they all said it's not relevant for any hunter spec, so this was dropped with 2.14.
> TLDR: it was broken, and wasn't used, now it's gone

I'd just like to add to Freeruns' statements; swing/shot timing is vital to hunters in classic. This may not be the case with retail versions, but in Vanilla/Classic the swing timer is THE paramount aura to track. It makes the entire addon difficult/pointless to use for the class.

**Comment by Freeruns** [comment]
And It is still the case with retail.
We need "timer started on HIT" it was not broken before and now it's gone...

**Comment by InfusOnWoW** [comment]
We did some changes that enabled the range swing time again.

**Comment by Freeruns** [comment]
Yes, thanks to Buds and Translit :D
You can check this wa from translit (it is work on HIT):  https://wago.io/RangedOnHitSwingTimer
